content/en/Getting started/planning.md

```diff
@@ -0,0 +1,31 @@
+---
+title: "Set Up Pentest Planning"
+linkTitle: "Set Up Pentest Planning"
+weight: 210
+description: >
+  Set up a schedule. Confirm the scope.
+---
+
+{{% pageinfo %}}
+Now you can set a date for the pentest.
+{{% /pageinfo %}}
+
+By default, we can help you schedule pentests with a start date at least
+48 hours after you select "Submit for Review".
+
+Normally, we complete our pentests in two (2) weeks.
+
+![Pentester Planning](/PentestPlanning.png "Schedule your pentest. Start at least 2 business days from today.")
+
+On the Pentest Planning screen, you can also update the asset scope, and
+therefore the number of credits required for the test.
+
+You can also specify special requirements for pentesters. For example, if
```

Comment:
I think the special requirements section is no longer in the wizard, and we are only enabling it through the Cobalt admin console. Not sure because there is also a feature flag involved.  There were recent code changes to remove it, you could verify with @marijanamustra and EX team.

Reply:
Hi, removing the requirements section from the wizard is under a feature flag so not out yet. The plan is that adding requirements will be available to CSMs in editing brief (and in admin under individual postings on the pentest). You can see more here https://zombie.atlassian.net/browse/EX-1183 or if you need more information, I would suggest posting a question on [EX channel](https://cobalthq.slack.com/archives/CTSGKCXAL) Thank you

Reply:
Thanks for the update. I did not know of the plan, and will monitor EX-1183. However, I need to reflect the Wizard as it exists today.